flume/src/main/java/es/tid/fiware/fiwareconnectors/cygnus/sinks/OrionMySQLSink.java

```diff
@@ -66,7 +66,7 @@
      */
     public OrionMySQLSink() {
         super();
-        logger = CygnusLogger.getLogger(OrionHDFSSink.class);
+        logger = CygnusLogger.getLogger(OrionMySQLSink.class);
```

Comment:
If the fix is meanifull, an entry should be added to CHANGES_NEXT_REELASE.

I don't understand what you mean by "tagging to release/ioteplatform-v1.1" but maybe we can dissuss that part out-of-band (by skype).